CONTRIBUTING.md

```diff
@@ -1,43 +1,38 @@
-Welcome to Findadoc Japan! We're grateful for our volunteers 😊
+Welcome to Find a Doc, Japan! We're grateful for our volunteers 😊
 
 # Where should I start?
 
-👉️ Join the [Slack](https://join.slack.com/t/find-a-doc/shared_invite/zt-s4744a6o-MGaGHzLN5wB9aXeha3vdsQ) 💬
+👉️ Join us on [Slack](https://join.slack.com/t/find-a-doc/shared_invite/zt-s4744a6o-MGaGHzLN5wB9aXeha3vdsQ) 💬
 
 👉️ Developers 💻️
 
-1. Fork the repo and build the source by following the [README](https://github.com/ourjapanlife/findadoc-frontend#readme)
-2. Find a good issue by checking out the [latest Project Board](https://github.com/ourjapanlife/findadoc-frontend/projects), or looking for something tagged "Good First Issue"
+1. Fork the repo and build the source by following the [README](./README.md)
+2. Find a good issue by checking out the [Issues Section](https://github.com/ourjapanlife/findadoc-web/issues), and look for something tagged "Good First Issue"
 3. Submit a PR and work with the team to merge it
    🙌🏻
+4. Give us a shout if you need any help on the `#frontend-team` channel on Slack
 
 👉️ Translators 🌐
 
-1. [Visit the Localization Subproject](https://github.com/ourjapanlife/findadoc-localization) for further instructions
+1. Join the `#localization` channel on Slack.
 
 👉️ UI/UX Designers (and people interested in developing new features) ✨
 
-1. Check out the Wiki](https://github.com/ourjapanlife/findadoc-frontend/wiki) to learn about User stories, etc.
-2. Join the #new-features Slack Channel
+1. Join the `#ui-ux-design` and `#frontend-team` channels on Slack
    🎨
 
-## What if I get stuck?
-
-- Developers: Check out the [TROUBLESHOOTING Guide](https://github.com/ourjapanlife/findadoc-frontend/wiki/Troubleshooting-%F0%9F%98%AD-%F0%9F%98%AD-%F0%9F%98%AD-%F0%9F%91%89%EF%B8%8F-%F0%9F%98%8A-%F0%9F%98%8A--%F0%9F%98%8A) or ask in Slack
-- Translators: Ask in slack in the #localization channel
-- Everyone: Ask in Slack
-
 ## Helpful Info
```

Comment:
should remove since empty for now, unless we're planning to add content here soon